src/fiskaltrust.Launcher/Commands/InstallLinuxSystemd.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Serilog;
+using System.Diagnostics;
+
+namespace fiskaltrust.Launcher.Commands
+{
+    public class InstallLinuxSystemd
```

Comment:
Generally, it might make sense to check the return values of the commands we call in this class, and print the output to the console in case it's != 0 - might be nice in case something fails during the process